overlord/hookstate/ctlcmd/get.go

```diff
@@ -57,36 +66,42 @@ Nested values may be retrieved via a dotted path:
 
     $ snapctl get author.name
     frank
+
+Values of plug or slot attributes may be printed in plug hooks via:
+
+    $ snapctl get :plugname serial-path
+	/dev/ttyS0
+
+    $ snapctl get --slot :slotname camera-path
+	/dev/video0
+
+Values of plug or slot attributes may be printed in slot hooks via:
+
+    $ snapctl get --plug :slotname serial-path
+	/dev/ttyS0
+
+    $ snapctl get :slotname camera-path
+	/dev/video0
 `)
 
 func init() {
-	addCommand("get", shortGetHelp, longGetHelp, func() command { return &getCommand{} })
+	addCommand("get", shortGetHelp, longGetHelp, func() command {
+		return &getCommand{}
+	}, map[string]string{
+		"slot": i18n.G("Access the slot side of the connection"),
+		"plug": i18n.G("Access the plug side of the connection"),
+	}, []argDesc{
+		{name: i18n.G("<snap>:<slot>")}})
```

Comment:
Same here. Also note that the "<snap>" part should be empty for "snapctl" we only support "local snap" on it for now.